### PR TITLE
Split url testing access access files

### PR DESCRIPTION
**Pull request by EliasBoulharts**

### CODE REVIEW

File: tests/permission_endpoint.py [hunk 1]
```diff
@@ -38,22 +39,60 @@ def __str__(self):
...
         return client.get(self.url, self.data)
 
 
-class TestingPostUIViews(TestingUIViews):
-    expected_status_code = 302
-
+class TestingPostContextView(TestingContextView):
     def call_url(self, client):
+        # if isinstance(client, APIClient):
+        #     return client.post(self.url, self.data, content_type="application/json")
```
Comment (a-belhadj) [inline tests/permission_endpoint.py:63]: still needed ?

File: tests/test_profiles/test_urls/test_quota.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                url_kwargs={'scope_id': self.test_quota_scope_team.id}
+            ),
+            TestingPostContextView(
+                url='profiles:team_quota_edit',
+                perm_str='profiles.change_quota',
+                url_kwargs={'scope_id': self.test_quota_scope_team.id},
+                data={
+                    f"attribute_definition_{self.cpu_attribute.id}": 100,
+                    f"attribute_definition_{self.other_attribute.id}": 50
+                }
```
Comment (a-belhadj) [inline tests/test_profiles/test_urls/test_quota.py:43]: Delete test is missing

File: tests/permission_endpoint.py (hunk 1 above)
Comment (EliasBoulharts) [reply tests/permission_endpoint.py:63]: Can be removed for now.

File: tests/test_profiles/test_urls/test_quota.py (hunk 2 above)
Comment (EliasBoulharts) [reply tests/test_profiles/test_urls/test_quota.py:43]: Delete is possible from UI? No test about before the split

File: tests/test_service_catalog/test_urls/test_support.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                perm_str='service_catalog.close_support',
+                url_kwargs={'instance_id': self.support_test.instance.id, 'pk': self.support_test.id},
+                expected_status_code=302
+            ),
+            TestingPostContextView(
+                url='service_catalog:support_close',
+                perm_str='service_catalog.close_support',
+                url_kwargs={'instance_id': self.support_test.instance.id, 'pk': self.support_test.id},
+                expected_status_code=405,
+                expected_not_allowed_status_code=405
```
Comment (a-belhadj) [inline tests/test_service_catalog/test_urls/test_support.py:75]: weird, how do we know that this view works ? We need to send a 403 when not allowed

File: tests/test_service_catalog/test_urls/test_support.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                perm_str='service_catalog.reopen_support',
+                url_kwargs={'instance_id': self.support_test.instance.id, 'pk': self.support_test.id},
+                expected_status_code=302
+            ),
+            TestingPostContextView(
+                url='service_catalog:support_reopen',
+                perm_str='service_catalog.reopen_support',
+                url_kwargs={'instance_id': self.support_test.instance.id, 'pk': self.support_test.id},
+                expected_status_code=405,
+                expected_not_allowed_status_code=405
```
Comment (a-belhadj) [inline tests/test_service_catalog/test_urls/test_support.py:88]: weird, how do we know that this view works ? We need to send a 403 when not allowed

**Review by a-belhadj — CHANGES_REQUESTED** [review]
Looks good, small changes to do. Thank Elias

File: tests/test_service_catalog/test_urls/test_support.py (hunk 4 above)
Comment (EliasBoulharts) [reply tests/test_service_catalog/test_urls/test_support.py:88]: We raise an exception for method not allowed before to check a permission. May this test must be moved elsewhere.

**Pull request merged**